src/symbolize/miri.rs

```diff
@@ -1,9 +1,18 @@
 use core::ffi::c_void;
 use core::marker::PhantomData;
 
-use crate::backtrace::miri::{resolve_addr, Frame};
-use crate::symbolize::{ResolveWhat, SymbolName};
-use crate::types::BytesOrWideString;
+use super::super::backtrace::miri::{resolve_addr, Frame};
+use super::BytesOrWideString;
+use super::{ResolveWhat, SymbolName};
+
+#[cfg(backtrace_in_libstd)]
+mod mystd {
+    pub use crate::*;
+}
+#[cfg(not(backtrace_in_libstd))]
+extern crate std as mystd;
```

Comment:
Instead of doing this, can the `filename` method be behind `#[cfg(feature = "std")]`? I don't think that the standard library uses that method.